test/e2e/framework/configmanager/constants.go

```diff
@@ -29,11 +29,13 @@ const (
 	// that contain logs, outputs etc
 	LogPerm = os.FileMode(0666)
 
-	YKScheduler   = "yunikorn-scheduler"
-	SchedulerName = "yunikorn"
-	YKAdmCtrl     = "yunikorn-admission-controller"
+	YKScheduler          = "yunikorn-scheduler"
+	YKSchedulerContainer = "yunikorn-scheduler-k8s"
+	SchedulerName        = "yunikorn"
+	YKAdmCtrl            = "yunikorn-admission-controller"
 	// YuniKorn service name
```

Comment:
As long as we are cleaning up, comment should go to the right (not above) the variable.